cortex/dataset/viewRGB.py

```diff
@@ -446,8 +448,9 @@ def color_voxels(channel1, channel2, channel3, channel1color, channel2color,
             blue.flat[i] = this_color[2]
 
         # Now make an alpha volume
-        alpha = np.ones_like(red, np.uint8) * 255
-        alpha[mask] = 0
+        if alpha is None:
+            alpha = np.ones_like(red, np.uint8) * 255
+            alpha[mask] = 0
```

Comment:
This should probably be unindented, in case there are NaNs in the red/green/blue arrays.